odoo/addons/base/models/ir_http.py

```diff
@@ -118,12 +120,28 @@ class IrHttp(models.AbstractModel):
     _name = 'ir.http'
     _description = "HTTP Routing"
 
+    @classmethod
+    def _slug(cls, value: models.BaseModel | str) -> str:
```

Comment:
it used to access a tuple, hinted by:
```py
def slug(value):
    try:
        ...
    except AttributeError:
        # assume name_search result tuple
        identifier, name = value
```
why doesn't it anymore? couldn't find the explanation in the commit message

Reply:
There is no longer/no case with using a tuple. I can put it back but it's dead code currently.